src/SlateAsInputEditor/index.js

```diff
@@ -39,7 +38,11 @@ const regex = /{{(.*?)}}/gm;
 
 const EditorWrapper = styled.div`
   background: #fff;
-  margin: 50px;
+  min-height: 750px;
+  max-width: ${props => props.width || null};
```

Comment:
I think it would make more sense to set max-width to the default property which is `none`. For css logic like this, let's try to stick to the default value as the alternative. https://www.w3schools.com/cssref/pr_dim_max-width.asp